src/integrationTest/resources/environment.properties

```diff
@@ -31,6 +31,7 @@ notifications.templates.email.defendantMoreTimeRequested = defendant-more-time-r
 frontend.base.url = https://civil-money-claims.co.uk
 oauth2.client.id = cmc_citizen
 oauth2.client.secret = 123456
+CASE_WORKER_ROLES=caseworker-cmc
```

Comment:
suggestion: as this specific to document management - should we define as `document_management.caseWorkerRoles` (seems consistent with others - although the underscore isn't!)

Reply:
happy to change it but not sure if this role will never be used for anything else.

Reply:
Please use dot(.) instead of underscore(_), in order to stay consistent